ydb/docs/en/core/concepts/topology.md

```diff
@@ -1,25 +1,104 @@
 
 {% note info %}
 
 Node failure means both its total and partial unavailability, for example, failure of a single disk on a node.
 
+{% endnote %}
+
+The table below describes the requirements and fault tolerance levels for different operating modes:
+
+| Mode | Storage<br>volume multiplier | Minimum<br>number<br>of nodes | Fail<br>domain | Fail<br>realm | Number of<br>data centers | Number of<br>server racks | Fault tolerance<br>level |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| `none` | 1 | 1 | Node | Node | 1 | 1 | No fault tolerance |
+| `block-4-2` | 1.5 | 8 (10 recommended) | Rack | Data center | 1 | 8 | Can stand a failure of 2 racks |
+| `mirror-3-dc` | 3 | 9 (12 recommended) | Rack | Data center | 3 | 3 in each data center | Can stand a failure of a data center and 1 rack in one of the two other data centers. |
+| `block-4-2`<br>(reduced) | 1.5 | 10 | ½ a rack | Data center | 1 | 5 | Can stand a failure of 1 rack. |
+| `mirror-3-dc`<br>(reduced) | 3 | 12 | ½ a rack | Data center | 3 | 6 | Can stand a failure of a data center and 1 server in one of the two other data centers. |
+| `mirror-3-dc`<br>(3 nodes) | 3 | 3 | Server | Data center | 3 | Doesn't matter | Can stand a failure of a single server, or a failure of a data center. |
+
+{% note info %}
+
 The storage volume multiplier specified above only applies to the fault tolerance factor. Other influencing factors (for example, slot fragmentation and granularity) must be taken into account for storage size planning.
 
 {% endnote %}
 
+When creating a [storage group](glossary.md#storage-groups), which is a basic allocation unit for storage management, {{ ydb-short-name }} chooses [VDisks](./glossary.md#vdisk) that are located on [PDisks](./glossary.md#pdisk) from different fail domains. For `block-4-2` mode, a storage group should be distributed across at least 8 fail domains, and for `mirror-3-dc` mode, across 3 fail realms, with at least 3 fail domains in each realm.
+
 For information about how to set the {{ ydb-short-name }} cluster topology, see [{#T}](../deploy/configuration/config.md#domains-blob).
+
+### Reduced configurations {#reduced}
+
+If it is impossible to use the [recommended amount](#cluster-config) of hardware, you can divide servers within a single rack into two dummy fail domains. In this configuration, the failure of one rack results in the failure of two domains instead of just one. In such reduced configurations, {{ ydb-short-name }} will continue to operate if two domains fail. The minimum number of racks in a cluster is five for `block-4-2` mode and two per data center (e.g. six in total) for `mirror-3-dc` mode.
+
+The minimal fault-tolerant configuration of a {{ ydb-short-name }} cluster uses the `mirror-3-dc-3-nodes` operating mode, which requires 3 servers. Each server in such configuration acts as both a fail domain and a fail realm, and a cluster can stand a failure of a single server only.
```

Comment:
```suggestion
If it is impossible to use the [recommended amount](#cluster-config) of hardware, you can divide servers within a single rack into two dummy fail domains. In this configuration, the failure of one rack results in the failure of two domains instead of just one. In such reduced configurations, {{ ydb-short-name }} will continue to operate if two domains fail. The minimum number of racks in a cluster is five for `block-4-2` mode and two per data center (e.g., six in total) for `mirror-3-dc` mode.

The minimal fault-tolerant configuration of a {{ ydb-short-name }} cluster uses the `mirror-3-dc-3-nodes` operating mode, which requires only three servers. In this configuration, each server acts as both a fail domain and a fail realm, and the cluster can withstand the failure of only a single server.
```